docs/stories/story-1.5.md

```
@@ -15,10 +15,17 @@ Draft
 
 ## Tasks / Subtasks
-- [ ] Implement the `get_portfolio` method in `src/ordo/adapters/fyers.py`.
-- [ ] Add logic to fetch data from all required Fyers endpoints.
-- [ ] Write the data mapping logic to transform the Fyers response into the standard `Portfolio` model.
-- [ ] Add error handling for Fyers API errors.
+- [x] Implement the `get_portfolio` method in `src/ordo/adapters/fyers.py`.
+- [x] Add logic to fetch data from all required Fyers endpoints.
+- [x] Write the data mapping logic to transform the Fyers response into the standard `Portfolio` model.
+- [x] Add error handling for Fyers API errors.
+
+## File List
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)